sdk/src/sysvar/mod.rs

```diff
@@ -1,13 +1,25 @@
 //! named accounts for synthesized data accounts for bank state, etc.
 //!
+#[cfg(feature = "kitchen_sink")]
 use crate::pubkey::Pubkey;
 
 pub mod clock;
+
+#[cfg(feature = "program")]
+pub mod clock_account_info;
+
+#[cfg(feature = "kitchen_sink")]
```

Comment:
All these `#[cfg(feature = "kitchen_sink")]` directives are making my 🍝 spidey sense tingle, but let's run this experiment for a little longer IMO...

Reply:
Yeah, agreed, let's see how it plays out